hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestHddsDispatcher.java

```diff
@@ -98,8 +100,10 @@ public void testContainerCloseActionWhenFull() throws IOException {
       Map<ContainerType, Handler> handlers = Maps.newHashMap();
       for (ContainerType containerType : ContainerType.values()) {
         handlers.put(containerType,
-            Handler.getHandlerForContainerType(containerType, conf, context,
-                containerSet, volumeSet, metrics));
+            Handler.getHandlerForContainerType(containerType, conf,
+                context.getParent().getDatanodeDetails().getUuidString(),
+                containerSet, volumeSet, metrics, c -> {
+                }));
```

Comment:
Nit: odd spacing for no-op consumer ([this one](https://github.com/apache/hadoop-ozone/pull/60/files#diff-d696db1fc042e05bbf8d305d5182f260R262) looks cleaner).  Same in a few other places.

Reply:
Hard to avoid this as this is the default formatting rule of IntelliJ. Will be reformatted by the next auto-format anyway.

But I created a constant field, hopefully it makes it a little more readable:

```
  public static final Consumer<ContainerReplicaProto> NO_OP_ICR_SENDER = c -> {};

```